slack/commands.py

```diff
@@ -352,6 +352,16 @@ def command_slack_debug(buffer: str, args: List[str], options: Options):
         print_uncaught_error(error, True, options)
 
 
+@weechat_command("-clean")
+def command_slack_status(buffer: str, args: List[str], options: Options):
+    status = args[0]
+    slack_buffer = shared.buffers.get(buffer)
```

Comment:
You have to check that `slack_buffer` is not None (it will be None if the command is run on a non-Slack buffer).